runtime/src/main/java/dev/ebullient/micrometer/runtime/MicrometerRecorder.java

```diff
@@ -26,13 +32,35 @@
     private static final Logger log = Logger.getLogger(MicrometerRecorder.class);
     static final int TRIM_POS = "quarkus.micrometer.export.".length();
 
+    private volatile static CompositeMeterRegistry quarkusRegistry;
+
+    public void createRootRegistry() {
+        Clock clock = Arc.container().beanManager().createInstance().select(Clock.class).get();
+        quarkusRegistry = new CompositeMeterRegistry(clock);
+        VertxMeterBinderAdapter.meterRegistry.set(quarkusRegistry);
+        CompositeRegistryCreator.rootRegistry = quarkusRegistry;
```

Comment:
Per comment above.. setting the CompositeRegistryCreator.rootRegistry in this way bugs me a little.